bin/serve.ts

```diff
@@ -1,45 +1,49 @@
-const express = require('express');
-require('dotenv').config();
+import type { ProjectInfo } from "@truffle/decoder";
+
+import { getProjectInfoForTransaction } from "../src";
+
+dotenv.config();
 
 const app = express();
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(bodyParser.json());
-app.get('/tx', handle);
+app.get("/tx", handle);
 console.log(`Starting on port ${process.env.PORT}`);
 
-async function handle (req, res) {
+async function handle(req, res) {
   // ?from=0x4b203f54429f7d3019c0c4998b88f8f3517f8352&to=0x28de5c5f56b6216441ee114e832808d5b9d4a775&data=0x67641c2f00000000000000000000000000000000000000000000000000000177b6486a7400000000000000000000000000000000000000000000000000000000602ed7f236372b070000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000080c886232e9b7ebbfb942b5987aa00000000000000000000000027054b13b1b798b345b591a4d22e6562d47ea75a0000000000000000000000000000000000000000000000000000000008f0d180000000000000000000000000000000000000000000000000000000000000000036372b07000000000000000000000000000000000000000000000000000000000000000000000000000000004b203f54429f7d3019c0c4998b88f8f3517f8352000000000000000000000000c02aaa39b223fe8d0a0e5c4f27ead9083c756cc2000000000000000000000000000000000000000000000000298a051db5f5ea00000000000000000000000000000000000000000000000000000000000000000036372b0700000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000008bb52b2f23008ba58939ff59a8f3f20000000000000000000000004572f2554421bd64bef1c22c8a81840e8d496bea0100000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001b5fcb0cc856bd0afc89493be7bb0e751a9b876b0faebe3086697b3c6c78e4efd3370a7eef528987c13555fd264d96b45af3277b555f9f4f4f6ebf9eb62d3fec2f
   const txParams = {
     from: req.params.from,
     to: req.params.to,
     input: req.params.data,
-    blockNumber: null,
-  }
+    blockNumber: null
+  };
 
   const config = TruffleConfig.default().merge({
     networks: {
-      'mainnet': {
-        url: process.env['MAINNET_URL'],
-        network_id: 1,
+      mainnet: {
+        url: process.env["MAINNET_URL"],
+        network_id: 1
       }
     },
-    network: 'mainnet',
+    network: "mainnet",
     // May need to make a more proper truffle config, will see. TODO
-    sourceFetchers: ['etherscan'],
+    sourceFetchers: ["etherscan"],
     etherscan: {
-      apiKey: process.env['ETHERSCAN_KEY'],
+      apiKey: process.env["ETHERSCAN_KEY"]
     }
   });
 
   try {
     const projectInfo: ProjectInfo = await getProjectInfoForTransaction({
       to: txParams.to,
-      networkId: '1',
-      config,
+      networkId: "1",
+      config
```

Comment:
Resisting having trailing comma debate.... My short reason is "it eliminates one-line diffs when manipulating the number of entries in an array/object"

Reply:
I have zero preference here. Add it to the prettier config if you like. I just copied Truffle's prettier config.